careers/devops.markdown

```diff
@@ -0,0 +1,33 @@
+---
+title: DevOps Engineer
+summary: The Haskell Foundation seeks a passionate DevOps professional to support GHC, related Haskell projects and tooling, and the HF itself.
+postedOn: 2021-12-28
+status: Open
+---
+
+# Job Details
+
+## Responsibilities
+
+* Maintain GitLab and CI infrastructure for GHC
+* Improve CI
+* Expand the CI infrastructure to include base the base Haskell ecosystem
+* Support other Haskell libraries and tools, including HLS (Haskell Language Server)
+* Work with the Haskell Infrastructure Team
+* Support OS packaging of GHC and tools
+* Documentation
+* Improvements and expansion of the GHC performance dashboard
+
+## Qualifications
+
+* Demonstrated ability to manage and simplify complexity
+* Collaborate with a variety of stakeholders
+* Can juggle multiple, conflicting priorities
+* Excellent communication skills
```

Comment:
```suggestion
* Excellent written communication skills
```

I would hope that there are also excellent oral communication skills, but our culture is strongly one of the written word.